app/controllers/invoice_items_controller.rb

```diff
@@ -0,0 +1,14 @@
+class InvoiceItemsController < ApplicationController
+  before_action :get_merchant
```

Comment:
I think you can add something like `,only :update` so it only calls this before action before the `update` action